util/vars.yml

```diff
@@ -2,92 +2,112 @@
 #
 # atm: `atmf???.nc` outputs
 # sfc: `sfcf???.nc` outputs
-control:
-  lat: 22.0
-  lon: 18.0
-  nz: 127
-  outname: '../tests/col.csv'
-  atmFile: 'atmf012.nc'
-  sfcFile: 'sfcf012.nc'
-  srcLoc: "/scratch1/NCEPDEV/stmp2/Barry.Baker/FV3_RT/rt_980816/atmaero_control_p8_intel"
+cases:
```

Comment:
@bbakernoaa can you check these are consistent with what you wanted?

Reply:
looks good